src/components/CallAssignmentList.spec.tsx

```diff
@@ -0,0 +1,61 @@
+import CallAssignmentList from './CallAssignmentList';
+import { mountWithProviders } from '../utils/testing';
+import {
+    ZetkinCallAssignment,
+    ZetkinMembership,
+} from '../types/zetkin';
+
+describe('CallAssignmentList', () => {
+    let dummyCallAssignments : ZetkinCallAssignment[];
+    let dummyMemberships : ZetkinMembership[];
+
+    beforeEach(()=> {
+        cy.fixture('dummyCallAssignments.json')
+            .then((data : {data: ZetkinCallAssignment[]}) => {
+                dummyCallAssignments = data.data;
+            });
+        cy.fixture('dummyMemberships.json')
+            .then((data : {data: ZetkinMembership[]}) => {
+                dummyMemberships = data.data;
+            });
+    });
+
+    it('contains data for each event', () => {
+        mountWithProviders(
+            <CallAssignmentList
+                callAssignments={ dummyCallAssignments }
+                memberships={ dummyMemberships }
+            />,
+        );
+
+        cy.get('[data-testid="call-assignment"]').each((item) => {
+            cy.wrap(item)
+                .get('[data-testid="call-title"]').should('be.visible')
+                .get('[data-testid="call-org"]').should('be.visible');
+        });
+    });
+
+    it('shows a placeholder when the list is empty', () => {
+        dummyCallAssignments = [];
+
+        mountWithProviders(
+            <CallAssignmentList
+                callAssignments={ dummyCallAssignments }
```

Comment:
Don't reassign the `dummyCallAssignments` variable. It's unnecessary, and will cause bugs if you switch to loading the fixtures `before()` instead of `beforeEach()`.

If you want a local exception to just this test, create a new variable, or better yet just pass `[]` directly to the `callAssignments` property, without using a variable at all.